CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.16.0-rc.0 / 2020-01-30
+
+* [CHANGE] PromQL: Remove MaxConcurrent from the PromQL engine opts #6712
+* [FEATURE] PromQL: Support trailing commas in grouping opts #6480
+* [FEATURE] PromQL: add absent_over_time query function #6490
+* [FEATURE] Adding optional logging of queries to their own file #6520
+* [ENHANCEMENT] Remote write: Improve sharding calculation in cases where we would always be consistently behind by tracking pendingSamples ##6511
+* [ENHANCEMENT] React UI: Add support for rules page and "Xs ago" duration displays #6503
+* [ENHANCEMENT] promtool: Add support for line-column numbers for invalid rules output #6533
+* [ENHANCEMENT] ReactUI: alerts page, replace filtering togglers tabs with checkboxes #6543
+* [ENHANCEMENT] PromQL: Use PositionRange in parser errors to improve error messages #6634
+* [ENHANCEMENT] TSDB: return an error on ingesting series with duplicate labels #6664
```

Comment:
Bugfix, though not technically user visible as we'll catch it in scrape first.

Reply:
No Brian

We don't catch in scrape, only when we go to tsdb. But scrape will fail.